lib/AST/ASTMangler.cpp

```diff
@@ -3192,6 +3192,14 @@ void ASTMangler::appendEntity(const ValueDecl *decl) {
   // Handle accessors specially, they are mangled as modifiers on the accessed
   // declaration.
   if (auto accessor = dyn_cast<AccessorDecl>(decl)) {
+    if (accessor->isDistributedThunk()) {
+      appendContextOf(decl);
+      appendDeclName(accessor->getStorage());
+      appendDeclType(accessor, FunctionMangling);
+      appendOperator("F");
```

Comment:
Distributed thunk accessors are mangled the same way as distributed funcs because runtime needs a function type, if we want to add a new mangling for accessor thunks we'd need to rework a runtime aspects of remote calls. I'm going to add a comment here explaining this.

Reply:
Added a comment about this.